Update password-policy-recommendations.md (#7796)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/misc/password-policy-recommendations.md

# Password policy recommendations

As the admin of an organization, you're responsible for setting the password policy for users in your organization. Setting the password policy can be complicated and confusing, and this article provides recommendations to make your organization more secure against password attacks.

> For Cloud Only Accounts Microsoft has a pre-defined password policy that can't be changed. The only item you can change is how many days until a password expires and whether or not passwords expire at all. 

Reviewer: ```suggestion
Microsoft cloud-only accounts have a pre-defined password policy that cannot be changed. The only items you can change are the number of days until a password expires and whether or not passwords expire at all. 
```
